MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include versioneer.py
+include xyzservices/_version.py
```

Comment:
Also include the license file while adding a manifest file?

Reply:
Good point. This is the one created automatically by versioneer. I'll add it.